pkg/controller/seed-controller-manager/mla/alertmanager_controller.go

```diff
@@ -55,18 +55,38 @@ const (
 	alertmanagerTenantHeaderName = "X-Scope-OrgID"
 )
 
-type alertmanagerReconciler struct {
+type alertmanagerController struct {
 	ctrlruntimeclient.Client
 	httpClient *http.Client
 
+	log                   *zap.SugaredLogger
+	cortexAlertmanagerURL string
+}
+
+func newAlertmanagerController(
+	client ctrlruntimeclient.Client,
+	log *zap.SugaredLogger,
+	httpClient *http.Client,
+	cortexAlertmanagerURL string,
+) *alertmanagerController {
+	return &alertmanagerController{
+		Client:     client,
+		httpClient: httpClient,
+
+		log:                   log,
+		cortexAlertmanagerURL: cortexAlertmanagerURL,
+	}
+}
```

Comment:
Can you please move line 58-79 to just above line 211? so we move the definition of `alertmangerController` near the member functions

Reply:
thx, done